release-plz.toml

```diff
@@ -0,0 +1,55 @@
+[changelog]
+body = """
```

Comment:
This generates changelog notes for new versions that look mostly like our existing format, except that it's further subdivided into "Added" (`feat` commits) and "Fixed" (`fixed`) commits.